services/app/src/hooks/useWsMessage.js

```diff
@@ -0,0 +1,19 @@
+export const useWsMessage = (ws) => {
```

Comment:
I dont think this should be in hooks folder. It is not a hook, its just event listener for ws messages.

Also, the name is not descriptive. This is a debug tool to track incomming messages. It should be named accordingly.